tests/kernel/fpu_sharing/generic/testcase.yaml

```diff
@@ -21,21 +21,21 @@ tests:
     tags: fpu kernel
     timeout: 600
   kernel.fpu_sharing.generic.riscv32:
-    extra_args: PI_NUM_ITERATIONS=500
+    extra_args: PI_NUM_ITERATIONS=500 DISABLE_INT_TEST=1
```

Comment:
What is the rationale behind disabling the timer testcase for these platforms?

Reply:
Those particular platforms fail the test.  Unfortunately, I don't enough about these platforms to fix them myself.  I updated the commit message to note this.

Reply:
Can you make 48b8439a0d4123d8abf83d08e170dbcc798e68cd a separate PR? This needs a more thorough review from the maintainers of all these platforms.